### ISSUE TITLE
Expect pure DOM directly

### ISSUE THREAD

**Issue by lijunle** [kevindente/expect-dom#2]
Do not want to wrap DOM to jQuery, or depends on jQuery. Any plan on this?

Something like this:

```js
var div = document.createElement('div');
expect(div).is.a('div')
```

**Comment by kevindente** [comment]
No plans at this time.

**Comment by lijunle** [comment]
@kevindente Hi, thanks for your reply.

I found that it has been almost 3y after last update. Is it welcome for a PR to support this? If yes, I could try to implement it.

**Comment by kevindente** [comment]
@lijunle yup, I'd definitely be open to that.

**Comment by lijunle** [comment]
Merged in #4
